op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -80,8 +80,8 @@ var (
 			common.HexToAddress("0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed"): false,
 		},
 	}
-	distributionScheduleAddressMap = map[uint64]common.Address{
-		AlfajoresNetworkID: common.HexToAddress("0x78af211ad79bce6bf636640ce8c2c2b29e02365a"),
+	unreleasedTreasuryAddressMap = map[uint64]common.Address{
+		AlfajoresNetworkID: common.HexToAddress("0x07bf0b2461A0cb608D5CF9a82ba97dAbA850F79F"),
```

Comment:
So it's not only renamed but also has a new address. I assume this is intentional and correct.

Reply:
Yes, the scope of the contract changes and it was deployed again.